From: David Paleino <dapal@debian.org>
Subject: migrate from node to node
Forwarded: not-needed
Reviewed-By Yadd <yadd@debian.org>
Last-Update: 2022-04-19

--- a/example/bool.js
+++ b/example/bool.js
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/node
 var util = require('util');
 var argv = require('optimist').argv;
 
--- a/example/boolean_double.js
+++ b/example/boolean_double.js
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/node
 var argv = require('optimist')
     .boolean(['x','y','z'])
     .argv
--- a/example/boolean_single.js
+++ b/example/boolean_single.js
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/node
 var argv = require('optimist')
     .boolean('v')
     .argv
--- a/example/default_hash.js
+++ b/example/default_hash.js
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/node
 
 var argv = require('optimist')
     .default({ x : 10, y : 10 })
--- a/example/default_singles.js
+++ b/example/default_singles.js
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/node
 var argv = require('optimist')
     .default('x', 10)
     .default('y', 10)
--- a/example/divide.js
+++ b/example/divide.js
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/node
 
 var argv = require('optimist')
     .usage('Usage: $0 -x [num] -y [num]')
--- a/example/line_count.js
+++ b/example/line_count.js
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/node
 var argv = require('optimist')
     .usage('Count the lines in a file.\nUsage: $0')
     .demand('f')
--- a/example/line_count_options.js
+++ b/example/line_count_options.js
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/node
 var argv = require('optimist')
     .usage('Count the lines in a file.\nUsage: $0')
     .options({
--- a/example/line_count_wrap.js
+++ b/example/line_count_wrap.js
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/node
 var argv = require('optimist')
     .usage('Count the lines in a file.\nUsage: $0')
     .wrap(80)
--- a/example/nonopt.js
+++ b/example/nonopt.js
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/node
 var argv = require('optimist').argv;
 console.log('(%d,%d)', argv.x, argv.y);
 console.log(argv._);
--- a/example/short.js
+++ b/example/short.js
@@ -1,3 +1,3 @@
-#!/usr/bin/env node
+#!/usr/bin/node
 var argv = require('optimist').argv;
 console.log('(%d,%d)', argv.x, argv.y);
--- a/example/string.js
+++ b/example/string.js
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/node
 var argv = require('optimist')
     .string('x', 'y')
     .argv
--- a/example/xup.js
+++ b/example/xup.js
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/node
 var argv = require('optimist').argv;
 
 if (argv.rif - 5 * argv.xup > 7.138) {
--- a/readme.markdown
+++ b/readme.markdown
@@ -29,7 +29,7 @@
 xup.js:
 
 ````javascript
-#!/usr/bin/env node
+#!/usr/bin/node
 var argv = require('optimist').argv;
 
 if (argv.rif - 5 * argv.xup > 7.138) {
@@ -56,7 +56,7 @@
 short.js:
 
 ````javascript
-#!/usr/bin/env node
+#!/usr/bin/node
 var argv = require('optimist').argv;
 console.log('(%d,%d)', argv.x, argv.y);
 ````
@@ -72,7 +72,7 @@
 bool.js:
 
 ````javascript
-#!/usr/bin/env node
+#!/usr/bin/node
 var util = require('util');
 var argv = require('optimist').argv;
 
@@ -101,7 +101,7 @@
 nonopt.js:
 
 ````javascript
-#!/usr/bin/env node
+#!/usr/bin/node
 var argv = require('optimist').argv;
 console.log('(%d,%d)', argv.x, argv.y);
 console.log(argv._);
@@ -123,7 +123,7 @@
 divide.js:
 
 ````javascript
-#!/usr/bin/env node
+#!/usr/bin/node
 var argv = require('optimist')
     .usage('Usage: $0 -x [num] -y [num]')
     .demand(['x','y'])
@@ -152,7 +152,7 @@
 default_singles.js:
 
 ````javascript
-#!/usr/bin/env node
+#!/usr/bin/node
 var argv = require('optimist')
     .default('x', 10)
     .default('y', 10)
@@ -169,7 +169,7 @@
 default_hash.js:
 
 ````javascript
-#!/usr/bin/env node
+#!/usr/bin/node
 var argv = require('optimist')
     .default({ x : 10, y : 10 })
     .argv
@@ -188,7 +188,7 @@
 boolean_single.js
 
 ````javascript
-#!/usr/bin/env node
+#!/usr/bin/node
 var argv = require('optimist')
     .boolean('v')
     .argv
@@ -205,7 +205,7 @@
 boolean_double.js
 
 ````javascript
-#!/usr/bin/env node
+#!/usr/bin/node
 var argv = require('optimist')
     .boolean(['x','y','z'])
     .argv
@@ -229,7 +229,7 @@
 line_count.js
 
 ````javascript
-#!/usr/bin/env node
+#!/usr/bin/node
 var argv = require('optimist')
     .usage('Count the lines in a file.\nUsage: $0')
     .demand('f')
--- a/test/_/bin.js
+++ b/test/_/bin.js
@@ -1,3 +1,3 @@
-#!/usr/bin/env node
+#!/usr/bin/node
 var argv = require('../../index').argv
 console.log(JSON.stringify(argv._));
--- a/test/parse.js
+++ b/test/parse.js
@@ -2,7 +2,7 @@
 var path = require('path');
 var test = require('tap').test;
 
-var $0 = 'node ./' + path.relative(process.cwd(), __filename);
+var $0 = '/usr/bin/node ./' + path.relative(process.cwd(), __filename);
 
 test('short boolean', function (t) {
     var parse = optimist.parse([ '-b' ]);
